core/src/main/java/org/openstack4j/model/compute/Keypair.java

```diff
@@ -35,5 +37,41 @@
 	 * @return the server fingerprint
 	 */
 	String getFingerprint();
+	
+	/**
+	 * 
+	 * @return the user_id for a keypair.
+	 */
+	String getUserId();
+	
+	/**
+	 * 
+	 * @return is deleted
+	 */
+	boolean getDeleted();
+	
+	/**
+	 * time created
+	 * @return
+	 */
+	Date getCreatedAt();
+	
+	/**
+	 * time updated
+	 * @return
+	 */
+	Date getUpdatedAt();
+	
+	/**
+	 * time deleted
+	 * @return
+	 */
+	Date getDeletedAt();
+	
+	/**
+	 * 
+	 * @return id of the keypair
+	 */
+	Integer getId();
```

Comment:
Looking at http://developer.openstack.org/api-ref/compute/?expanded=create-or-import-keypair-detail#create-or-import-keypair - What's the `id` of the keypair?